.github/workflows/bindings-drift.yml

```
@@ -0,0 +1,44 @@
+        with:
+          node-version: 18
+          cache: 'npm'
+          cache-dependency-path: frontend/package-lock.json
+
+      - name: Install frontend dependencies
+        working-directory: frontend
+        run: npm ci
+
+      - name: Generate contract bindings
+        env:
+          CONTRACT_ID: ${{ secrets.CONTRACT_ID || format('cat contracts/contract_id.txt') }}
```

actionlint (1.7.12)

[error] number of arguments is wrong. function "format(string, any...) -> string" takes at least 2 parameters but 1 arguments are given

(expression)